zebra/zebra_rib.c

```diff
@@ -3214,6 +3239,71 @@ unsigned long rib_score_proto_table(uint8_t proto, unsigned short instance,
 	return n;
 }
 
+/*
+ * Delete remaining zebra_originationg routes from previous run of zebra
+ * after kernel reconciliation.
+ */
+void rib_sweep_kernel_reconcile_table(struct route_table *table)
```

Comment:
This code in this function is exactly same as rib_sweep_table() function except the check for RECONCILE FLAG. Can we consider merging these two with appropriate flags ?

Reply:
Sure, If reviewer agrees, I am up to combine them, I feel the same. To start with, I wanted to keep kernel_reconcile code separate, so started with separate function. Thanks

Reply:
Please merge.

Reply:
yes let's merge them.